Kickstarter-iOS/Views/RewardCardView.swift

```diff
@@ -160,6 +159,7 @@ public final class RewardCardView: UIView {
     self.pillCollectionView.rac.hidden = self.viewModel.outputs.pillCollectionViewHidden
     self.rewardTitleLabel.rac.hidden = self.viewModel.outputs.rewardTitleLabelHidden
     self.rewardTitleLabel.rac.text = self.viewModel.outputs.rewardTitleLabelText
+    self.includedItemsTitleLabel.rac.textColor = self.viewModel.outputs.sectionTitleLabelTextColor
```

Comment:
Just noticed here that the output name doesn't match the element that's being set?

Reply:
Good catch!